services/web/server/src/simcore_service_webserver/catalog_handlers_reverse_proxy.py

```diff
@@ -36,10 +36,15 @@ async def reverse_proxy_handler(request: web.Request) -> web.Response:
     # FIXME: hack
     if "/services" in backend_url.path:
         backend_url = backend_url.update_query({"user_id": user_id})
-    logger.debug("Redirecting '%s' -> '%s'", request.url, backend_url)
+    logger.debug(
+        "Redirecting '%s' -> '%s'",
+        request.url,
+        backend_url,
+        extra=get_extra({"user_id": user_id}),
```

Comment:
And this is how I would use it
```suggestion
        extra=get_extra(user_id=user_id),
```

Reply:
Thanks I like it good idea :+1: will do in next commit